assets/js/blocks/active-filters/block.tsx

```diff
@@ -413,15 +412,18 @@ const ActiveFiltersBlock = ( {
 	);
 };
 
-ActiveFiltersBlock.propTypes = {
+interface ActiveFiltersBlock {
 	/**
 	 * The attributes for this block.
 	 */
-	attributes: PropTypes.object.isRequired,
+	attributes: {
+		// Define the properties of the attributes object.
+		// Adjust the types accordingly.
+	};
```

Comment:
I noticed that on https://github.com/woocommerce/woocommerce-blocks/blob/trunk/assets/js/blocks/active-filters/types.ts, we already have type definitions. We could reuse them in this PR. On https://github.com/hritikchaudhary/woocommerce-blocks/blob/923f89ef0a2050438d84fd2b4097f8c84d1e0ad5/assets/js/blocks/product-category/block.tsx#L13 and https://github.com/hritikchaudhary/woocommerce-blocks/blob/923f89ef0a2050438d84fd2b4097f8c84d1e0ad5/assets/js/blocks/product-category/block.tsx#L28, you can find an example of how to accomplish this.